src/components/CreateCredential.js

```diff
@@ -17,53 +20,131 @@ const CreateCredential = () => {
+      <GetCredential.UnknownError />
 
-        <GetCredential.Side>
-          <div className='side-container'>
+      <GetCredential.Card title="Your credential is ready to use" paragraph="Check the downloads section of your browser for the ZIP file, or find it where you save downloads on your machine." nextStepsLabel="Next steps" nextStepsHref="/credentials/nextsteps" className="card_developer_console">
+
+        <GetCredential.Card.Side>
+          <div style="display : flex ; gap : 16px ; flex-direction : column;">
             <h3 className="spectrum-Heading spectrum-Heading--sizeS side-header" >API key credential</h3>
             <p className="spectrum-Body spectrum-Body--sizeM">An API Key credential was created. The API key credential identifies your application to Adobe servers and can help accept or reject request originating from certain domains.</p>
             <h3 className="spectrum-Heading spectrum-Heading--sizeS side-header" >Learn more</h3>
-            <a className="side-documentation" href='https://developer.adobe.com/express/embed-sdk/docs/guides/quickstart/'>Quickstart guide</a>
-            <a className="side-documentation" href='https://developer.adobe.com/express/embed-sdk/docs/guides'>Adobe Express Embed SDK documentation</a>
+            <a className="side-documentation" style={{ color: "#0265DC" }} href='https://developer.adobe.com/express/embed-sdk/docs/guides/quickstart/'>Quickstart guide</a>
+            <a className='side-documentation' style={{ color: "#0265DC" }} href='https://developer.adobe.com/express/embed-sdk/docs/guides/'>Adobe Express Embed SDK documentation</a>
           </div>
-        </GetCredential.Side>
+        </GetCredential.Card.Side>
+
+        <GetCredential.Card.Products label="Included products and services">
+          <GetCredential.Card.Product label="Adobe Express Embed SDK" icon={creativeCloud} />
+        </GetCredential.Card.Products>
+
+        <GetCredential.Card.DevConsoleLink heading="Developer Console Project" />
+
+        <GetCredential.Card.CredentialDetails heading="Credential details">
+          <GetCredential.Card.CredentialDetails.APIKey heading="API Key" />
+          <GetCredential.Card.CredentialDetails.AllowedOrigins heading="Allowed domains" />
+          <GetCredential.Card.CredentialDetails.OrganizationName heading="Organization" />
+        </GetCredential.Card.CredentialDetails>
 
       </GetCredential.Card>
 
+      <GetCredential.Return title="Previously created projects" paragraph="Select a project and access your existing credentials for Firefly - Firefly and Creative Cloud Automation." nextStepsLabel="Next steps" nextStepsHref="/credentials/nextsteps" className="card_developer_console">
```

Comment:
Shouldn't it say `Adobe Express Embed SDK` instead of `Firefly - Firefly and Creative Cloud Automation`?

Reply:
My bad , I'll update